mp/src/game/server/momentum/mom_generic_bomb.h

```diff
@@ -0,0 +1,37 @@
+﻿//========= Copyright Valve Corporation, All rights reserved. ============//
+//
+// Purpose: Exploding bomb
+//
+//=============================================================================//
+
+#ifndef CMomGenericBomb_H
+#define CMomGenericBomb_H
+#ifdef _WIN32
+#pragma once
+#endif
```

Comment:
You don't need to do define guards anymore, every compiler we use supports just using `#pragma once`, no need for these `ifndef` and `_WIN32` checks